Source/SwiftLintFramework/Rules/Lint/EmptyMethodOverrideRule.swift

```diff
@@ -0,0 +1,158 @@
+import Foundation
+import SourceKittenFramework
+
+public struct EmptyMethodOverrideRule: SubstitutionCorrectableASTRule,
+ConfigurationProviderRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    static let nonTriggeringExamples = [
+        """
+        override func viewDidDisappear(_ animated: Bool) {
+            childViewController.viewDidDisappear(animated)
+        }
+        """,
+        """
+        override func viewDidDisappear(_ animated: Bool) {
+            super.viewDidDisappear(animated)
+            print("View controller did disappear")
+        }
+        """,
+        """
+        public override init() {
+            super.init()
+        }
+        """,
+        """
+        override func setUp() {
+            super.setUp()
+            urlString = "https://httpbin.org/basic-auth"
+        }
+        """
+    ].map(wrapInClass)
+
+    static let triggeringExamples = [
+        """
+        override func a(){/*comment*/super.a()}
+        """,
+        """
+        override func viewDidLoad() {
+            super.viewDidLoad()
+
+            // Do any additional setup after loading the view.
+        }
+        """,
+        """
+        override func didReceiveMemoryWarning() {
+            super.didReceiveMemoryWarning()
+            // Dispose of any resources that can be recreated.
+        }
+        """,
+        """
+        override func becomeFirstResponder() -> Bool {
+            return super.becomeFirstResponder()
+        }
+        """,
+        """
+        internal
+        class
+        override
+        func setUp() {
+            super.setUp()
+        }
+        """
+    ].map(wrapInClass)
+
+    static let corrections = Dictionary(uniqueKeysWithValues:
+        EmptyMethodOverrideRule.triggeringExamples.map { ($0, wrapInClass("")) })
```

Comment:
Clearly preferred:

```swift
    static let corrections = Dictionary(
        uniqueKeysWithValues: EmptyMethodOverrideRule.triggeringExamples.map { ($0, wrapInClass("")) }
    )
```
or by using a variable like this:

```swift
    static let corrections: [String: String] = {
        let keysWithValues = EmptyMethodOverrideRule.triggeringExamples.map { ($0, wrapInClass("")) }
        return Dictionary(uniqueKeysWithValues: keysWithValues)
    }()
```

Reply:
I agree